src/core/diags/README.md

```diff
@@ -222,6 +222,26 @@ Return the state of the radio.
 sleep
 ```
 
+### diag radio enable
+
+Enable radio interface and put it in receive mode.
+
+```bash
+> diag radio enable
+Radio link started.
+sleep
```

Comment:
Where does the `sleep` come from? Could we add the `Done` at the end of the command output?